parser-sdk/nodejs/parser-wrapper.js

```diff
@@ -119,12 +119,18 @@ async function main() {
   const resultUploadUrl = process.argv[3];
 
   console.log("Fetching result file");
-  const { data } = await axios.get(resultFileUrl);
+  let response;
+  if(scan.spec.scanType === "amass"){
```

Comment:
I'd prefer to put something like a `encodingType` property on the ParseDefinition which can then specify if this is supposed to be a binary file or to be parsed as a plaintext file.
Right now all generic components, lurker and operator are completely generic and independent of the scanner, this would be the first point (of which I'm aware) where a central component knows a detail of a scanner.